dist/info/mame2000_libretro.info

```diff
@@ -10,4 +10,4 @@ license = "MAME"
 permissions = ""
 display_version = "0.37b5"
 supports_no_game = "false"
-notes = "- BIOS files go into the ROM directory|"
+notes = "(!) The BIO files must be inside the ROM directory.|"
```

Comment:
I wouldn't change on account of translations.
I'm sticking with this for now. Everything is nice and translation-friendly.

Then you know what? It's not even being translated at all... Oh well.

Good thing you found this. It's the only one saying BIO, not BIOS. Thanks. :-)